src/CommonITILObject.php

```diff
@@ -3473,6 +3473,10 @@ public static function getNewStatusArray()
         return $tab;
     }
 
+    public static function getProcessStatusArray()
+    {
+        // To be overriden by class
```

Comment:
Should probably return an empty array.
```suggestion
        // To be overriden by class
        return [];
```